agent/azure-monitor-exporter/src/main/java/com/azure/monitor/opentelemetry/exporter/implementation/pipeline/TelemetryItemExporter.java

```diff
@@ -3,6 +3,7 @@
 
 package com.azure.monitor.opentelemetry.exporter.implementation.pipeline;
 
+import static com.azure.monitor.opentelemetry.exporter.implementation.utils.AksResourceAttributes.OTEL_RESOURCE_ATTRIBUTES;
```

Comment:
it might be nice to inject otelResourceAttributes into this class, then the TelemetryItemExporterTest can inject whatever it wants instead of having to rely on the global AksResourceAttributes state

Reply:
this has been moved to AksResourceAttributes and i came up a reloading for testing purposes.